src/pages/DirectMessage.tsx

```diff
@@ -201,9 +251,40 @@ const DirectMessage = () => {
               <InfoOutlinedIcon />
             </IconButton>
           </Box>
-          <Divider />
+
           <Box sx={styles.chatContainer}>
             <List component="div" ref={messageRef} sx={styles.messageList}>
+              <Box
+                sx={styles.bioContainer}
+                onClick={() => navigate(`/${selectedConversation.username}`)}
+              >
+                {loading ? (
+                  <CircularProgress />
+                ) : (
+                  <>
+                    <Box sx={styles.bioDetails}>
+                      <Avatar sx={styles.avatar} />
+                    </Box>
+                    <Box sx={styles.nameContainer}>
+                      <Typography variant="h6" sx={styles.displayName}>
+                        {selectedConversation.displayName}
+                      </Typography>
+                      <Typography variant="body2">
+                        {`@${selectedConversation.username}`}
+                      </Typography>
+                    </Box>
+                    <Typography>{selectedConversation.bio}</Typography>
+                    <Typography variant="body2">
+                      {`Joined ${formatTimestamp(
+                        selectedConversation.joinedDate ?? ""
+                      )} •
```

Comment:
since I need to render both the joinedDate and followerCount what about something like this?

```
{selectedConversation.joinedDate &&
                        selectedConversation.followerCount &&
                        `Joined ${formatTimestamp(
                          selectedConversation.joinedDate
                        )} •
                        ${selectedConversation.followerCount} Followers`}
```

Reply:
As discussed offline, it doesn't really make sense to have an undefined joinedDate, so we need to always render that. I see you've made the changes tho, nice 👍

I made a small syntax adjustment to help readability